worker/ci/runner/docker.go

```diff
@@ -38,6 +38,9 @@ import (
 const (
 	// BindTemplate is the template of binds.
 	BindTemplate = "%s:%s"
+
+	// Flag of running a container which use the built image as a service.
+	BuiltImage = "BUILT_IMAGE"
```

Comment:
// BuiltImage is the keyword of the image built in the build step.

is more appropriate?